tsconfig.json

```diff
@@ -21,6 +21,7 @@
     /* Module Resolution Options */
     "moduleResolution": "node16",          /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
     "types": ["node"],                     /* Type declaration files to be included in compilation. */
+    "resolveJsonModule": true,
```

Comment:
```suggestion
    "resolveJsonModule": true,             /* Allows importing modules with a .json extension. Helpful for version check.*/
```

Reply:
Done in 753c58e9677cbec1007f9bbefcb0d36f44f8a5e2